internal/entity/user.go

```diff
@@ -8,27 +8,27 @@ type User struct {
 	Name           string    `json:"name"`
 	Email          string    `json:"email"`
 	PermissionRole role.Role `json:"roles"`
-	PasswordHash   []byte    `json:"-"`
+	PasswordHash   string    `json:"-"`
 }
 
-// NewUser represent new user in request.
-type NewUser struct {
-	ID              string `json:"id" validate:"omitempty"`
-	Name            string `json:"name,omitempty" validate:"omitempty"`
-	Email           string `json:"email" validate:"required,email"`
-	Password        string `json:"password" validate:"required"`
-	PasswordConfirm string `json:"password_confirm" validate:"omitempty,eqfield=Password"`
-}
+// // NewUser represent new user in request.
+// type NewUser struct {
+// 	ID              string `json:"id" validate:"omitempty"`
+// 	Name            string `json:"name,omitempty" validate:"omitempty"`
+// 	Email           string `json:"email" validate:"required,email"`
+// 	Password        string `json:"password" validate:"required"`
+// 	PasswordConfirm string `json:"password_confirm" validate:"omitempty,eqfield=Password"`
+// }
 
-type PasswordsRequest struct {
-	UserID          string `json:"id"`
-	Password        string `json:"password"`
-	PasswordConfirm string `json:"password_confirm"`
-}
+// type PasswordsRequest struct {
+// 	UserID          string `json:"id"`
+// 	Password        string `json:"password"`
```

Comment:
oh, I used it for my functions, do you really need to delete it?

Reply:
it wos my mistake to add it,

Reply:
we should separate http and usecase and usecase should do same for other cases than http

Reply:
yep. then we will wait for changes of your second branch. Need to find a boilerplaite generator for mappers

Reply:
but what mappers do we need  ?